### ISSUE TITLE
An error occurred. Please try again. If the problem persists, please contact the site administrator.

### ISSUE THREAD

**Issue by nyedlapalli** [microsoft/sample-app-aoai-chatGPT#111]
After deploying app I am getting below error.

**An error occurred. Please try again. If the problem persists, please contact the site administrator.**

**Comment by sarah-widder** [comment]
Hi @nyedlapalli can you share more details about your set up? Did you set all of the required environment variables correctly for your app?

**Comment by nyedlapalli** [comment]
@sarah-widder Yes, created new app service with Python and deployed code. perfectly worked in one app. but trying to deploy in different Subscriptions I am getting this error. tried multiple accounts and multiple apps. but same error.

**Comment by redcrxsi** [comment]
This happened to me as well when I deployed today, though my deployment was right from Azure AI Studio. I did check all the configuration values for accuracy, and they all are correct. I'll keep an eye on things for changes or new builds..

**Comment by sarah-widder** [comment]
Hi @nyedlapalli @redcrxsi there was an issue with the static files checked into the repo where some of the error handling code was out of date. This has now been fixed. 

For apps deployed manually:
- Make sure to build the frontend with the latest code or pull the static files merged in `main`, then redeploy.

For apps deployed from the Azure AI Studio:
- Locate the app service resource in the Azure Portal, go to the Deployment Center, and click 'Sync' to pull in the fix from yesterday. When the sync completes, restart the app.

**Comment by redcrxsi** [comment]
Thanks @sarah-widder much appreciated - I did see that, however I deployed with de20819, it was a brand-new site earlier today and the logs show a commit ID of de20819. I went ahead and tried your suggestion though, sync, redeployment and restart however the behavior remains the same.

If there's additional data I can gather for you please let me know I'd be happy to collect anything!

**Comment by fellnerse** [comment]
Same here, error persists

**Comment by jayjaysn** [comment]
Unfortunately I still have the same error. The error does not appear in our logs either. Is there any new information? Also the idea of @sarah-widder to apply the static folder fix absolutely does not solve the problem

**Comment by pamelafox** [comment]
@jayjaysn Can you find the full logs? Open the App Service app in the Portal, find Advanced Tools in the left hand nav, open that tools website up, then click "Download as zip" on that page. When the zip downloads, open it up and find the logs named "*_default_docker.txt". Scroll down in those to find the latest logs.

**Comment by jayjaysn** [comment]
> @jayjaysn Can you find the full logs? Open the App Service app in the Portal, find Advanced Tools in the left hand nav, open that tools website up, then click "Download as zip" on that page. When the zip downloads, open it up and find the logs named "*_default_docker.txt". Scroll down in those to find the latest logs.

Please excuse the misunderstanding. I already know how to get the logs. The problem is that there is no warning or error in the logs that would help me to correct the initial error "An error occurred. Please try again. If the problem persists, please contact the site administrator". Therefore I turn here to you...

**Comment by redcrxsi** [comment]
@jayjaysn seems like your issue is different, if you can't access your OpenAI resource through other means then I would think that needs to be resolved first.

Personally, I don't have that issue, my OpenAI resource works fine both directly through an Azure Logic App via API and through the Azure OpenAI Studio.

It's just this deployment that doesn't work. Log files via Kudu show basically nothing, 1 single line which is odd.

2023-08-01T10:26:53.930Z INFO  - 

Is there some kind of debug logging flag I can enable or should I try using the Log Stream and reproduce?

**Comment by davidm00** [comment]
Hi, is it possible for you to check the network tab in your browser's console to see what the error response is exactly. This error could also be different based on whether you have applied a data source or not. I would first try to see exactly what the error message is in the dev tools network tab. (right click -> click inspect -> use the >> arrow and/or click network tab) Here you can see the failed response and further inspect the body. In some regions, there may be high traffic issues that could have limited/throttled the resource and given that response. There also could be an issue with the env variables (typo or incorrect value).
